db_implementation/split_json.py

```diff
@@ -1,27 +1,53 @@
+# author: tyler osborne
+# tyler.osborne@stonybrook.edu
+
 import orjson
 import time
+import pprint
+import glob
+import os
 
 if __name__ == '__main__':
+    print("Splitting MPQA JSON into bite-sized pieces...")
     start = time.time()
-    # f = open('mpqa.json', 'r', encoding='utf-8')
-    # content = f.read()
-    # js = orjson.loads(content)
-    # f.close()
-    # del content
-
-    f = open('mpqa_csds.json', 'r', encoding='utf-8')
-    content = orjson.loads(f.read())
-    first = content[:len(content) // 2]
-    second = content[(len(content) // 2) + 1:]
+    pp = pprint.PrettyPrinter()
+
+    # removing existing files
+    for f in glob.glob("mpqa_*.json"):
+        os.remove(f)
+
```

Comment:
Do you need to delete the files before you replace them?  Can you not simply overwrite them?

Reply:
Or is this to delete the old first and second half files?  If so, shouldn't that only ever happen once?  Whereas it's conceivable that we'd have to run the script more than once — I guess if Ghasem's group comes out with a revision or something.

Reply:
it is to delete the old first and second files, yes.

Reply:
My point is that you should do that manually, since it's  not part of the possibly repeated workflow.